### PR TITLE
Clean up repo and allow returning a result to the browser

### PR DESCRIPTION
**Pull request by C-Loftus**
@jaresty I commented out function calling, cleaned up a few older things in the repo, and added `browser` as an insertionMethod so we can say something like `model explain this browser` and have it explain the text in a new window in the browser.

### CODE REVIEW

File: .github/.markdown-link-check.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "timeout": "10s",
+    "retryOn429": true,
+    "retryCount": 5,
+    "fallbackRetryDelay": "30s",
+    // We have to treat 403 as success, because some sites have ddos protection. So we just need to manually check those
```
Comment (jaresty) [inline .github/.markdown-link-check.json:6]: It looks like // isn't valid in json?

Comment (C-Loftus) [reply .github/.markdown-link-check.json]: yup fixing this

**Comment by C-Loftus** [comment]
@jaresty if this looks good feel free to merge

File: GPT/staticPrompt.talon-list [hunk 2]
```diff
@@ -34,11 +33,9 @@ convert to yam: Convert the following data into a yaml format.
...
-describe summary: Condense the following text into a summary that is less verbose but still gives me the gist of the content. Do not describe the formatting.
-check grammar: Check the grammar of the following text. Return a list of all potential errors.
+check grammar: Check the grammar and formatting of the following text. Return a list of all potential errors.
 check spelling: Check the spelling of the following text. Return a list of all potential errors.
-check structure: I want you to skim the structure and layout of the following text. Tell me if the structure and order of my writing is correct. If it is not correct or flows poorly then tell me what might be wrong with it. If it is all correct then say it looks good. Do not describe the formatting. Do not comment on the specific content only its ordering and layout.
+check structure: Skim the structure and layout of the following text. Tell me if the structure and order of my writing is correct. If it is not correct or flows poorly then tell me what might be wrong with it. If it is all correct then say it looks good. Do not describe the formatting.
```
Comment (jaresty) [inline GPT/staticPrompt.talon-list:38]: I read somewhere that negative assertions like "do not describe" do not work as well as positive ones like "describe only the structure"

Comment (C-Loftus) [reply GPT/staticPrompt.talon-list:38]: Yup so isn't this fine then? I removed that negative assertion

Comment (jaresty) [reply GPT/staticPrompt.talon-list:38]: I think there's still a negative assertion here: 'Do not describe the formatting.'

Comment (C-Loftus) [reply GPT/staticPrompt.talon-list]: fixed

**Review by jaresty — APPROVED** [review]
I don't have time to try it right now but it looks good to me

**Pull request merged**